### PR TITLE
Fix no css on load error

### PR DESCRIPTION
**Pull request by sunilunka**
Have fixed the load css error that was occuring a stylesheet has no css. 
Also refined style mode menus and the mode itself so if the user loads a page, clears a grid, loads a grid or creates something new, style mode is reset.

### CODE REVIEW

File: browser/js/common/factories/GridFactory.js [hunk 1]
```diff
@@ -160,6 +160,7 @@ app.factory('GridFactory', function($http, $compile, PageFactory, ProjectFactory
         };
         PageFactory.savePage(page._id, changes)
             .then(function(updatedPage) {
+                console.log("UPDATED PAGE: ", updatedPage);
```
Comment (codebyuma) [inline browser/js/common/factories/GridFactory.js]: Is this console log still needed?

**Comment by codebyuma** [comment]
Code looks ok to me. Need to do more user testing once we have it all in.

File: browser/js/common/directives/styleNestedGridItem/styleNestedGridItem.directive.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+app.directive("styleNestedGridItem", function(StyleModeFactory){
+  return {
+    restrict: "A",
+    template: "<span class='glyphicon glyphicon-pencil'></span>",
+    link: function(scope, element, attrs){
+
+      var styleThisElement = false;
+
+      element.on("click", function(event){
+        console.log("ELEMENT ID: ", $(element).data("element-selector"));
```
Comment (sarahedkins) [inline browser/js/common/directives/styleNestedGridItem/styleNestedGridItem.directive.js]: remove console.log

Comment (sarahedkins) [inline browser/js/common/directives/styleNestedGridItem/styleNestedGridItem.directive.js]: remove console.log

**Pull request merged**